src/main/java/de/tum/in/www1/artemis/service/scheduled/cache/monitoring/ExamCache.java

```diff
@@ -0,0 +1,55 @@
+package de.tum.in.www1.artemis.service.scheduled.cache.monitoring;
+
+import com.hazelcast.config.*;
+import com.hazelcast.core.HazelcastInstance;
+
+import de.tum.in.www1.artemis.config.Constants;
+import de.tum.in.www1.artemis.service.scheduled.cache.Cache;
+import de.tum.in.www1.artemis.service.scheduled.cache.CacheHandler;
+
+final class ExamCache extends CacheHandler<Long> {
+
+    public ExamCache(HazelcastInstance hazelcastInstance) {
+        super(hazelcastInstance, Constants.HAZELCAST_MONITORING_CACHE);
+    }
+
+    /**
+     * Configures Hazelcast for the ExamCache before the HazelcastInstance is created.
+     *
+     * @param config the {@link Config} the ExamCache-specific configuration should be added to
+     */
+    static void configureHazelcast(Config config) {
+        ExamMonitoringCache.registerSerializers(config);
+        // Important to avoid continuous serialization and de-serialization and the implications on transient fields
+        // of ExamMonitoringCache
+        EvictionConfig evictionConfig = new EvictionConfig().setEvictionPolicy(EvictionPolicy.NONE);
+        NearCacheConfig nearCacheConfig = new NearCacheConfig() //
```

Comment:
What's up with all the `//` at the end of the lines here?

Reply:
I guess this was intended to enforce line breaks. But i use the `@formatter:off` instead :+1: